sdk/core/core-lro/test/utils/testTokenCredential.ts

```diff
@@ -0,0 +1,21 @@
+import { TokenCredential, GetTokenOptions, AccessToken } from "@azure/core-http";
```

Comment:
This file (and all copies of it) needs a copyright header:

```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

```

Reply:
I've pushed a commit to add the copyright for the new file added to core-arm
None of the files in core-lro has the copyright text, so logged a issue for it #5614